<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page_3">
      <widget class="QWidget" name="widget_4" native="true">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>481</width>
         <height>381</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>481</width>
         <height>381</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>481</width>
         <height>381</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_title">
          <property name="font">
           <font>
            <family>仿宋</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="text">
           <string>欢迎来到**云盘~</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
          <property name="textInteractionFlags">
           <set>Qt::LinksAccessibleByMouse</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>50</number>
           </property>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <family>Adobe Arabic</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>用户名：</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>15</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_username">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>123456</string>
             </property>
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="leftMargin">
            <number>50</number>
           </property>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>Adobe Arabic</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>密码：</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>30</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLineEdit" name="edit_password">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>123456</string>
             </property>
             <property name="maxLength">
              <number>16</number>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="button_login">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>登录</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="button_register">
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string>注册</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5">
      <widget class="QTableWidget" name="FriendsList">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>60</y>
         <width>421</width>
         <height>512</height>
        </rect>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>200</number>
       </attribute>
       <column>
        <property name="text">
         <string>好友名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>消息摘要</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="AddFriends">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>160</y>
         <width>151</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>添加好友</string>
       </property>
      </widget>
      <widget class="QPushButton" name="StartChat">
       <property name="geometry">
        <rect>
         <x>570</x>
         <y>290</y>
         <width>151</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>黑体</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>开始对话</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="session">
            <property name="text">
             <string>好友列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="upload">
            <property name="text">
             <string>上传</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="download">
            <property name="text">
             <string>下载</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="share">
            <property name="text">
             <string>分享</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="text">
             <string>传输列表</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="fresh">
            <property name="text">
             <string>刷新目录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_user_name">
            <property name="text">
             <string>用户：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Button_logout">
            <property name="text">
             <string>登出</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>fresh</zorder>
         <zorder>upload</zorder>
         <zorder>download</zorder>
         <zorder>share</zorder>
         <zorder>pushButton_5</zorder>
         <zorder>label_user_name</zorder>
         <zorder>Button_logout</zorder>
         <zorder>session</zorder>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="files">
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>28</number>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>200</number>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>28</number>
         </attribute>
         <row>
          <property name="text">
           <string>1</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>文件名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>文件大小</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>上传时间</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QWidget" name="widget_6" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>返回文件目录</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="transport">
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>200</number>
         </attribute>
         <column>
          <property name="text">
           <string>文件名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>文件大小</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>传输进度</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QTableWidget" name="files_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>751</width>
         <height>511</height>
        </rect>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>28</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>200</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>28</number>
       </attribute>
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>文件名</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>文件大小</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>上传时间</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string/>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="shareConfirm">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>530</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
      <widget class="QPushButton" name="shareCancel">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>530</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
